int16[2]	            ball_pos
int16[10]       	    yellow_team_pos
int16[5]    	        yellow_team_orientation
int16[10]       	    blue_team_pos
int16[5]    	        blue_team_orientation
uint16                  vision_fps